base/special/exp.jl

```diff
@@ -136,3 +147,36 @@ function exp(x::T) where T<:Union{Float32,Float64}
         return T(1.0) - ((x*p)/(p - T(2.0)) - x)
     end
 end
+
+# Method
+# 1. Argument reduction: Reduce x to an r so that |r| <= ln(2)/512. Given x,
+#    find r and integers k, j such that
+#       x = k*ln(2) + j/256 + r,  0 <= j < 256, |r| <= ln(2)/512.
+#
+# 2. Approximate exp(r) by it's degree 3 taylor series around 0.
+#    Since the bounds on r are very tight, this is sufficient to be accurate to floating point epsilon.
+#
+# 3. Scale back: exp(x) = 2^k * 2^(j/256) * exp(r)
+#    Since the range of possible j is small, 2^(j/256) is simply stored for all possible values.
+#    Technichally the J_table stores a scaled version which makes subnormal numbers work better.
+
+
+function myexp(x::T) where T<:Float64
```

Comment:
Calling it `myexp` is convenient for me copy/pasting to test...

Reply:
The only reason for that was that T is shorter, so it saves some characters in the conversion.